src/components/DegreeChipStrip.css

```
@@ -206,36 +224,18 @@
-  font-size: 0.82rem;
-}
+.app-container[data-layout-tier="mobile"] .degree-chip-strip-header { font-size: 0.92rem; }
+.app-container[data-layout-tier="mobile"] .degree-chip-note        { font-size: 1.1rem; }
+.app-container[data-layout-tier="mobile"] .degree-chip-interval    { font-size: 0.82rem; }
 
-/* Phase 2: Desktop compact chip strip — smaller chips, tighter padding.
-   Targets ~82px strip height (down from ~104px), keeping chip glyph legible. */
+/* Desktop: smaller chips, tighter padding (~82px strip height). */
 .app-container[data-layout-tier="desktop"] .degree-chip-strip {
   --chip-size: clamp(1.82rem, 2.45vw, 2.02rem);
   padding: 0.34rem 0.78rem 0.38rem;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)